### PR TITLE
Enable and apply Spotless

### PR DESCRIPTION
**Pull request by wborn**
Let's enable and apply Spotless.

Replaces https://github.com/openhab/openhab-addons/pull/8068

The issues Spotless found in the `feature.xml` are resolved by making sure the `openhab_wst_feature_file.prefs` include config matches.

I suggest to do follow up PRs for any further improvements.
That way we can quickly merge this PR and it doesn't turn into a merge conflict magnet. :magnet: :wink:

### CODE REVIEW

**Review by fwolter — APPROVED** [review]

File: bundles/org.openhab.binding.lutron/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -94,7 +94,8 @@
 				<label>Turn On To Last Level</label>
 				<description>
 					If set to true, dimmer will go to the last non-zero level set when an ON command is received.
-					If the last level cannot be determined, the value of onLevel will be used instead.
+					If the
+					last level cannot be determined, the value of onLevel will be used instead.
```
Comment (bobadair) [inline bundles/org.openhab.binding.lutron/src/main/resources/ESH-INF/thing/thing-types.xml:98]: I don't understand why it made this change, The line isn't as long as the one above it, and the break was inserted in a very odd place.

Comment (wborn) [reply bundles/org.openhab.binding.lutron/src/main/resources/ESH-INF/thing/thing-types.xml:98]: Yes I also noticed that and have now created #8413 to improve the formatting. It seems to format well when all text is on one line but seems to bug when there is still room left on previous lines.

**Comment by kaikreuzer** [comment]
@wborn It seems this broke our sandbox/release build since the version plugin changes the poms in a way that spotless complains, see https://ci.openhab.org/view/Sandbox/job/sandbox-openhab2-release/1634/consoleFull.

What can we do about that?

**Comment by wborn** [comment]
The quick fix is probably adding `-Dspotless.check.skip=true` to some release steps.

A nicer fix would be if the pipeline wouldn't touch the formatting or would run spotless:apply so we don't have to manually do this after every release. WDYT @pfink?

**Pull request merged**